doc/plugins/eu.esdihumboldt.hale.doc.user/toc.xml

```diff
@@ -118,6 +118,9 @@
          <topic href="html/tasks/alignment/remove_cell.html" label="Deleting a relation">
          </topic>
       </topic>
+      <topic label="Map">
```

Comment:
Is it not possible to create the parent topic in the map help bundle?
Like this, if the map help is not included, the Map tasks topic will be there, but empty.